src/features/duplicates/components/PotentialDuplicateModal.tsx

```diff
@@ -20,7 +20,7 @@ const PotentialDuplicateModal: FC<Props> = ({
   const { orgId } = useNumericRouteParams();
   const { mergeDuplicate } = useDuplicatesMutations(orgId);
 
-  return (
+  return open ? (
```

Comment:
I think this is a fine solution, although I would have phrased it like this (without a ternary):

```suggestion
  return open && (
```

There is also a more general problem with conditionally including the element (regardless of phrasing) which is that animations won't fully work.

I think an even better (but more complex) solution would have been to have an internal component for just the content inside of the modal that is conditionally included based on `open`. But that's way less important than fixing the bug, so I will merge this as is. There will be refactors in the future where the animation issue can be resolved.